presto-main/src/main/java/com/facebook/presto/sql/gen/LambdaBytecodeGenerator.java

```diff
@@ -67,6 +77,33 @@ private LambdaBytecodeGenerator()
     {
     }
 
+    public static Map<LambdaDefinitionExpression, CompiledLambda> generateMethodsForLambda(
```

Comment:
It looks like you can unify the duplicated `generateMethodsForLambda` into this one if you add a prefix parameter here. You don't need to do this in this PR. But I think that should happen.

Reply:
I agree. Will do this separately.